packages/app-bridge/src/types/Document.ts

```diff
@@ -96,23 +106,35 @@ export type DocumentApi = Simplify<
 
 export type Document = CamelCasedPropertiesDeep<DocumentApi>;
 
-export type DocumentLink = Simplify<Document & DocumentAsLink & { linkSettings: { newTab: boolean } }>;
+export type DocumentLink = Simplify<Document & DocumentAsLink>;
 
 export type DocumentLibrary = Simplify<
     Document & DocumentAsNoneLink & { settings?: { project: number }; mode: DocumentLibraryMode }
 >;
 
 export type DocumentStandard = Simplify<Document & DocumentAsNoneLink>;
 
+export type DocumentLinkSettings = CamelCasedPropertiesDeep<DocumentLinkSettingsApi>;
+
+export enum FormLinkStateType {
```

Comment:
There are no Form in this context ;)
```suggestion
export enum LinkSettingsDisplay {
```